### PR TITLE
python deps for grafana

### PR DESCRIPTION
**Pull request by jgphpc**
Needed for grafana.

eb ./Python-bare-2.7.12.eb 
== temporary log file in case of crash /dev/shm/piccinal/easybuild/stage/tmp/eb-4y55M5/easybuild-Rs3qjB.log
== processing EasyBuild easyconfig /apps/common/UES/sandbox/jgp/production.git/easybuild/easyconfigs/p/Python-bare/Python-bare-2.7.12.eb
== building and installing Python-bare/2.7.12...
== fetching files...
== creating build dir, resetting environment...
== unpacking...
== patching...
== preparing...
== configuring...
== building...
== testing...
== installing...
== taking care of extensions...
== installing extension setuptools 28.3.0 (1/8)...
== installing extension pip 9.0.1 (2/8)...
== installing extension nose 1.3.7 (3/8)...
== installing extension virtualenv 15.1.0 (4/8)...
== installing extension readline 6.2.4.1 (5/8)...
== installing extension six 1.10.0 (6/8)...
== installing extension influxdb 4.0.0 (7/8)...
== installing extension inflow 0.2.0 (8/8)...
== postprocessing...
== sanity checking...
== cleaning up...
== creating module...
== permissions...
== packaging...
== COMPLETED: Installation ended successfully
== Results of the build can be found in the log file(s) /users/piccinal/easybuild/daint/haswell/software/Python-bare/2.7.12/easybuild/easybuild-Python-bare-2.7.12-20170217.143829.log
== Build succeeded for 1 out of 1
== Temporary log file(s) /dev/shm/piccinal/easybuild/stage/tmp/eb-4y55M5/easybuild-Rs3qjB.log* have been removed.
== Temporary directory /dev/shm/piccinal/easybuild/stage/tmp/eb-4y55M5 has been removed.

### CODE REVIEW

**Comment by jgphpc** [comment]
will redo with a bare + external for my extensions

**Comment by gppezzi** [comment]
Python-bare will anyway conflict with the Python3 used by the regression, please open an issue on PyRegression to clarify this before proceeding with this PR.

**Comment by jgphpc** [comment]
correct. I removed Python-bare from this PR and using external module now.

**Comment by gppezzi** [comment]
This cannot be loaded together with the PyRegression module (cannot depend on python 2&3 at the same time, even if you don't see the conflict in the module file because we use Python-bare).

Do you want it to merge it anyway and use this independently from PyRegression?

**Comment by jgphpc** [comment]
I will try to build influx with Python3, seems to be supported.

**Comment by jgphpc** [comment]
https://madra.cscs.ch/scs/PyRegression/issues/196

File: easybuild/easyconfigs/i/influxdb/influxdb-4.0.0-CrayGNU-2016.11-Python-3.5.2.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    ('Python', '3.5.2'),
+]
+
+exts_list = [
+    ('influxdb', '4.0.0', {
+    'source_urls': ['https://pypi.python.org/packages/source/i/influxdb'],
+    }),
+    ('inflow', '0.2.0', {
+    'source_urls': ['https://pypi.python.org/packages/source/i/inflow'],
+    'patches': ['inflow-0.2.0.patch'],
```
Comment (gppezzi) [inline easybuild/easyconfigs/i/influxdb/influxdb-4.0.0-CrayGNU-2016.11-Python-3.5.2.eb]: Please add one comment on why you need the patch

**Comment by jgphpc** [comment]
## Weird character  
* grep licensed README.rst |cat -v

```
Inflow is licensed under `Mozilla Public License`_. M-BM-) 2016 `Advanced Climate
```

mll daint-gpu
mll Python/3.5.2-CrayGNU-2016.11
python setup.py build

```
Traceback (most recent call last):
  File "setup.py", line 7, in <module>
    long_description = open(os.path.join(cwd, 'README.rst'), 'r').read()
  File "/apps/daint/UES/jenkins/6.0.UP02/gpu/easybuild/software/Python/3.5.2-CrayGNU-2016.11/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 1462: ordinal not in range(128)
```

**Comment by gppezzi** [comment]
Sorry I meant a (one-line) comment directly on the .eb file, so that we know in the future why the patch is needed. Something like 
```
# Fixes problem with non-ascii character
```

**Pull request merged**